nistats/reporting/glm.py

```diff
@@ -0,0 +1,411 @@
+"""
```

Comment:
OK, so now, you split nistats.reporting in a bunch of small files. tell me if I'm wrong, but basically, there is nothing new below.

Reply:
Yeah. Couldn't stick all the new code in one reporting.py